hummingbird/ml/operator_converters/sklearn/linear.py

```diff
@@ -30,7 +30,7 @@ def convert_sklearn_linear_model(operator, device, extra_config):
     """
     classes = [0] if not hasattr(operator.raw_operator, "classes_") else operator.raw_operator.classes_
 
-    if not all([type(x) in [int, np.int32, np.int64] for x in classes]):
+    if not all(["int" in str(type(x)) for x in classes]):
```

Comment:
was this a bug,  or related to pipelines?  (if not, maybe make a separate PR next time for refactorings to avoid confusion?)

Reply:
This was something crashing some pipeline test on win machines (no idea why TBH).